pages/common/go-env.md

```diff
@@ -0,0 +1,16 @@
+# go env
+
+> Tool for printing Go environment information.
+> More information: <https://golang.org/cmd/go/>.
+
+- Show all go environment information:
+
+`go env`
+
+- Show specific environment variable:
+
+`go env {{GOPATH}}`
+
+- Set go environment value:
```

Comment:
Set an environment value. Also let's use a concrete example like "{{GOBIN}}={{path/to/directory}}"